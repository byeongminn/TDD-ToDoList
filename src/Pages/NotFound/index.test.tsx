/*
  <NotFound 페이지 컴포넌트 사양>
  1. 화면에 "Not Found"라는 글자를 표시한다.
*/

import { render, screen } from '@testing-library/react';
import { NotFound } from '.';

describe('<NotFound />', () => {
  it('renders component correctly', () => {
    const { container } = render(<NotFound />);

    const message = screen.getByText('Not Found');
    expect(message).toBeInTheDocument();

    expect(container).toMatchSnapshot();
  });
});
