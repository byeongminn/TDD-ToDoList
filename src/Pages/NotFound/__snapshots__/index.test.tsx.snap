// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders component correctly 1`] = `
<div>
  <div
    class="sc-aYaIB bYdvGL"
  >
    Not Found
  </div>
</div>
`;
